absorption_modalaSean.py

```diff
@@ -176,128 +164,130 @@ def smooth(norm_flux, box_size):
+                    obs_wavelength_OIfinal_index = np.min (np.where (wavelength>obs_wavelength_OIfinal))
+                    obs_wavelength_OI_final_vel = beta_list[obs_wavelength_OIfinal_index]
+                    plt.axvspan(obs_wavelength_OI_vel,obs_wavelength_OI_final_vel, alpha = 0.2, color = 'yellow')
+
+                    BI_ind_sum = round(sum(BI_ind), 2)
+                    BI_individual.append(BI_ind_sum) # this array contains one single BI value of each absortopn feature in a single spectrum
                     BI_ind = []
                     
-                    EW_ind_sum = round(sum(EW_ind),2)
+                    EW_ind_sum = round(sum(EW_ind), 2)
                     EW_individual.append(EW_ind_sum)
                     EW_ind = []
                                     
-                    final_depth=round((1.-np.min(normalized_flux[vmaxs_index:vmins_index])),2)
+                    final_depth = round((1. - np.min(normalized_flux[vmaxs_index:vmins_index])), 2)
                     final_depth_individual.append(final_depth)
-                    print('depth',final_depth_individual)
+                    print('depth', final_depth_individual)
                     
-                    count2=0                     
+                    count2 = 0                     
                                         
         else: #if the brac value is not more than zero (so if we don't have absorption feature)
-            part=0 # this is so b/c we do not want to keep counting the width of the absorption feature if it is not wider than 600km/s
-            count2=0# this is so b/c if the code encounters an other absorption feature which is wider than 600km/s, the code is going to go through the if statement on line 205
-            EW_ind=[]
+            part = 0 # this is so b/c we do not want to keep counting the width of the absorption feature if it is not wider than 600km/s
+            count2 = 0 # this is so b/c if the code encounters an other absorption feature which is wider than 600km/s, the code is going to go through the if statement on line 205
+            EW_ind = []
         
         if value == min_limit_value:
-            BI_total= round(sum(BI_mid),2)         
+            BI_total = round(sum(BI_mid),2)         
             BI_all.append(BI_total)    
             BI_all_individual.append(BI_individual)
             EW_all_individual.append(EW_individual)
    
-    if (len(vmaxs) != 0) or (plotall == 'yes'): 
-        yes=(str(file_count)+';' + str(spectrum_file_name) + '\n' + 'BI (' + str(max_velocity_limit) + ' > v > ' + str(min_velocity_limit) +'): ' + str(BI_total) + '\n' 
-             +  'vmins: ' + str(vmins) + '\n' + 'vmaxs: '+str(vmaxs) + '\n' + 'BI_individual: '+ str(BI_individual) + '\n' + 'EW_individual: '+ str(EW_individual) + '\n' 
-             + 'Depth: '+ str(final_depth_individual) +'\n'+'\n')
-        vlast.append(yes)
+    if (len(vmaxs) != 0) or (plotall == 'yes'):
```

Comment:
addressed my own comment from the last pull request
it's the only real change in this pull request (everything else is just white space)
I think it looks more readable this way